static/app/components/codecov/integratedOrgPicker/integratedOrgPicker.tsx

```diff
@@ -0,0 +1,36 @@
+import {useSearchParams} from 'react-router-dom';
+
+import {useCodecovContext} from 'sentry/components/codecov/context/codecovContext';
+import {t} from 'sentry/locale';
+
+import {
+  IntegratedOrgSelector,
+  type IntegratedOrgSelectorProps,
+} from './integratedOrgSelector';
+
+export interface IntegratedOrgProps extends Partial<IntegratedOrgSelectorProps> {}
+
+export function IntegratedOrgPicker({
+  triggerProps = {},
+  ...selectProps
+}: IntegratedOrgProps) {
+  const {integratedOrg} = useCodecovContext();
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  return (
+    <IntegratedOrgSelector
+      triggerLabel={integratedOrg ?? t('Select an integrated organization')}
```

Comment:
On my last pass, I tried trimming down the selector/picker component, so I moved props into the selector component unless I intended them to be dynamic. Thoughts on moving the `triggerLabel`, `triggerProps` (I didn't need it in the end) and `chosenOrg` into the selector itself? Leaving only the onChange really.

Actually makes me wonder if we even need the Picker wrapper at all...